Neos.Flow.Log/Tests/Unit/Backend/AbstractBackendTest.php

```diff
@@ -32,9 +32,9 @@ protected function setUp(): void
         $this->backendClassName = 'ConcreteBackend_' . md5(uniqid(mt_rand(), true));
         eval('
 			class ' . $this->backendClassName . ' extends \Neos\Flow\Log\Backend\AbstractBackend {
```

Comment:
interesting piece of code, maybe we should replace it with a test implementation of the abstract backend at some point.